docs/developer/quickstart.md

```diff
@@ -107,3 +107,33 @@ make lint
 
 > **Note**
 > fieldalignment errors can be fixed by running: `fieldalignment -fix <path-to-package>`
+
+## Run the Helm Linter
+
+If you have made changes to the Helm Chart, run the following make command from the project's root directory to lint the
+Helm Chart code:
+
+```shell
+make lint-helm
+```
+
+## Run go generate
+
+To ensure all the generated code is up to date, run the following make command from the project's root directory:
+
+```shell
+make generate
+```
+
+## Update NJS module ConfigMaps
+
+If you have made changes to the [NJS module code](/internal/mode/static/nginx/modules/README.md), run the following make
```

Comment:
I think this section needs to be rephrased too. Something like "To update the NJS ConfigMap yaml..."